lexfloatclient/LexFloatClient.h

```diff
@@ -269,6 +269,18 @@ LEXFLOATCLIENT_API int LF_CC DropFloatingLicense();
 */
 LEXFLOATCLIENT_API int LF_CC HasFloatingLicense();
 
+/*
+    FUNCTION: RequestOfflineFloatingLicense()
+
+    PURPOSE: Sends the request to lease the license from the LexFloatServer for offline usage.
+
```

Comment:
@muneebkq Parameter is missing in the docs. Also doc for this function is not complete.

Reply:
added